packages/desktop-client/src/browser-preload.browser.js

```diff
@@ -22,7 +22,7 @@ function createBackendWorker() {
     type: 'init',
     version: ACTUAL_VERSION,
     isDev: IS_DEV,
-    hash: process.env.REACT_APP_BACKEND_WORKER_HASH
+    hash: process.env.REACT_APP_BACKEND_WORKER_HASH || 'dev'
```

Comment:
Running `yarn start` does not set this env variable by default. There probably is a more eloquent way to get this working, so I'm open to suggestions.

But we NEED to fix this to make the local dev server work via `yarn start`.